lib/shim/shim.js

```diff
@@ -633,7 +645,9 @@ function wrapReturn(nodule, properties, spec, args) {
         return true
       },
       set: function setTrap(target, key, val) {
-        if (key === symbols.unwrap) {
+        if (key === symbols.wrapped) {
```

Comment:
put a comment explaining this.  since this is a proxy we don't want to propagate the wrapped symbol to the original but instead just keep it on `this` the wrapped "thing"